doozer/doozerlib/cli/scan_osh.py

```diff
@@ -64,6 +68,60 @@ def __init__(self, runtime: Runtime, last_brew_event: int, dry_run: bool, check_
         # Initialize JIRA client
         self.jira_client: JIRA = self.runtime.build_jira_client()
 
+    # Static functions
+    @staticmethod
+    def is_jira_workflow_component_disabled(meta):
+        """
+        Check if the OCPBUGS workflow is enabled in ocp-build-data image meta config
+        """
+        flag = meta.config.external_scanners.sast_scanning.jira_integration.enabled
+
+        # Disable only if specifically defined in config
+        return flag in [False, "False", "false", "no"]
+
+    def is_jira_workflow_group_disabled(self):
```

Comment:
```suggestion
    def is_jira_workflow_group_enabled(self):
```
This to me is more intuitive, since the default is to keep it `disabled`

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/247/commits/3153a81eca090e9df7930ea6c7fa6546572b4596